ci-operator/config/openshift/ironic-agent-image/openshift-ironic-agent-image-main.yaml

```diff
@@ -26,6 +26,8 @@ raw_steps:
       rm -f /etc/yum.repos.d/ci-rpm-mirrors.repo
       #use rhel8.6 beta repos
       curl http://base-4-11-rhel86.ocp.svc > /etc/yum.repos.d/base-4-11-rhel86.repo
+      #NOTE(elfosardo): this should be disabled globally
```

Comment:
Any reason to not reintroduce your change? TRT reverted as a matter of policy, not because the change wasn't worthwhile.  The criteria to get back in are [in the revert](https://github.com/openshift/release/pull/27980#issue-1209601262).

Reply:
This change is a bit different from the last ones, we are only applying this to the ironic images